### PR TITLE
Add FOSDEM 2020.

### PR DESCRIPTION
**Pull request by jonsger**
FOSDEM is coming :)

### CODE REVIEW

**Comment by loomchild** [comment]
Thanks a lot for a PR - I will test and merge it over the weekend. 

See you @ FOSDEM!

File: setup.py [hunk 1]
```diff
@@ -39,6 +39,8 @@
         ('share/sojourner/conferences/fosdem2018', ['conferences/fosdem2018/fosdem2018.cfg']),
         ('share/sojourner/conferences/fosdem2019', ['conferences/fosdem2019/banner.png']),
         ('share/sojourner/conferences/fosdem2019', ['conferences/fosdem2019/fosdem2019.cfg']),
+        ('share/sojourner/conferences/fosdem2020', ['conferences/fosdem2020/banner.png']),
+        ('share/sojourner/conferences/fosdem2020', ['conferences/fosdem2020/fosdem2019.cfg']),
```
Comment (loomchild) [inline setup.py]: There's a little mistake, it should be fosdem2020.cfg, I'll fix it.

**Review by loomchild — COMMENTED** [review]
Looks good, thanks. There's a little typo, I will fix it and make a new release and test in with N900 later today.

**Comment by jonsger** [comment]
Thanks, we need a script for this update process :P

**Comment by loomchild** [comment]
Hehe, perhaps you'd like to become a maintainer? 

BTW, the 0.3.8 release doesn't seem to work, there's an issue vith TLS version. I have set-up a HTTP proxy and are testing a new version.

**Comment by loomchild** [comment]
Now it should with version 0.3.9.

**Pull request merged**